### PR TITLE
Feature/baip 1569 add permanentId to span decomposition

### PR DESCRIPTION
**Pull request by mumuchuchu**

### CODE REVIEW

File: src/schema/span-installation/junction-box.repository.ts [hunk 1]
```diff
@@ -173,6 +174,7 @@ export class JunctionBoxRepository implements IJunctionBoxRepository {
 						...junctionBox,
 						id: newJunctionBoxId,
 						surveyId,
+						permanentId: ovsSurveyId,
```
Comment (fortunato) [inline src/schema/span-installation/junction-box.repository.ts:177]: I think the value should be the junctionBox.id and not the surveyId, the surveyId is already stored on this record.
If we use the id of the original junctionBox we can connect all potential clones of the original junctionBox and also work out what survey each belongs to (original or cloned).

File: src/schema/span-installation/models/junction-box.model.ts [hunk 2]
```diff
@@ -66,4 +66,9 @@ export class JunctionBox {
 
 	@Field((type) => String, { nullable: true })
 	deletedAt: string;
+
+	// For contract 3 (spanMeasures) survey: This is a reference for the surveyId that the decompositions are coloned from.
```
Comment (fortunato) [inline src/schema/span-installation/models/junction-box.model.ts:70]: Not the survey Id but the id of the junctionBox in contract 1

**Review by fortunato — CHANGES_REQUESTED** [review]

File: src/schema/span-installation/junction-box.repository.ts (hunk 1 above)
Comment (mumuchuchu) [reply src/schema/span-installation/junction-box.repository.ts]: Ok

File: src/schema/span-installation/models/junction-box.model.ts (hunk 2 above)
Comment (mumuchuchu) [reply src/schema/span-installation/models/junction-box.model.ts]: Ok

**Review by fortunato — APPROVED** [review]

**Pull request merged**